admin/check_tag_version_match.py

```diff
@@ -0,0 +1,31 @@
+#
+# Used during the release process to make sure that we release based on a
+# tag that has the same version as the current twisted.__version.
+#
+# Designed to be conditionally called inside GitHub Actions release job.
+#
+# To be called as: admin/check_tag_version_match.py refs/tags/twisted-20.3.0
+#
+import sys
+from twisted import __version__
```

Comment:
you might want to use `pep517.meta.load(...).version`

Reply:
done. thanks.